SaaSHelperFunctions.ps1

```diff
@@ -27,4 +27,6 @@
     }
 
     return $bcAuthContext, $headers, $endPointURL
-}
\ No newline at end of file
+}
+
+Export-ModuleMember -Function Create-SaasUrl
```

Comment:
The Create-SaasUrl should definitely not be exported - it is in the helperfunctions, which should be available to the functions inside containerhelper

Reply:
And it shouldn't be in the .psd1 either.
All other functions use this function and it is not needed for them.

Reply:
okay, sorry - then I misunderstood. I have adjusted the files accordingly.